neuralmonkey/config/configuration.py

```diff
@@ -67,11 +63,40 @@ def load_file(self, path):
             log("Failed to load INI file: {}".format(exc), color='red')
```

Comment:
Podle mne je to tam proto, aby se do logu přidalo "Failed to load INI file", ale nepsal jsem to, nevím. Zatím bych to nechal, vyřeší se to v dalším stadiu refaktorizace.